src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,74 @@
-export const Pagination = () => {};
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onChangePage: (page: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onChangePage,
+}) => {
+  const pagesAmount = Math.ceil(total / perPage);
+  const pages = new Array<number>(pagesAmount).fill(0).map((_, i) => i + 1);
+
+  const prevPage = () => {
+    if (currentPage > 1) {
+      onChangePage(currentPage - 1);
+    }
+  };
+
+  const nextPage = () => {
+    if (currentPage < pagesAmount) {
+      onChangePage(currentPage + 1);
+    }
+  };
+
+  return (
+    <ul className="pagination">
+      <li className={`page-item ${currentPage === 1 ? 'disabled' : ''}`}>
```

Comment:
it' better to use a library for conditional class names
Example:
https://www.npmjs.com/package/classnames